src/c++/vdo/fake/linux/blk_types.h

```diff
@@ -15,9 +15,22 @@
 struct bio;
 typedef u32 __bitwise blk_opf_t;
 typedef unsigned int blk_qc_t;
-typedef uint32_t blk_status_t;
+typedef u8 __bitwise blk_status_t;
 typedef void (bio_end_io_t) (struct bio *);
 
+#define BLK_QC_T_NONE -1U
+
+/* 
+ * Block error status values. See block/blk-core:blk_errors for details.
+ */
+#define BLK_STS_OK 0
+#define BLK_STS_NOSPC           ((blk_status_t)3)
+#define BLK_STS_RESOURCE        ((blk_status_t)9)
+#define BLK_STS_IOERR           ((blk_status_t)10)
+
+/* hack for VDO, don't use elsewhere */
+#define BLK_STS_VDO_INJECTED    ((blk_status_t)31)
+
 /*
```

Comment:
Why do we need to duplicate this block? Can't this just include the uds blkdev.h header?